amip_interp/include/amip_interp.inc

```diff
@@ -418,7 +80,7 @@ if ( .not.use_daily ) then
    if ( .not. allocated(temp2) ) allocate (temp2(mobs,nobs))
 
    if (use_zonal) then
-      call zonal_sst (Amip_Time, sice, temp1)
+      call ZONAL_SST_ (Amip_Time, sice, temp1)
```

Comment:
Nice, this seems to be better than declaring bunch of generic interfaces for private procedures